command/test-fixtures/push-tfvars/main.tf

```diff
@@ -7,7 +7,6 @@ variable "baz" {
 
   default = {
     "A"    = "a"
-    interp = "${file("t.txt")}"
```

Comment:
@jen20 I noticed this was here. We've always had validation that this shouldn't work (interpolations not allowed as variable defaults). This was passing when it shouldn't have. Is this okay to remove? It doesn't work anyways.